spec/support/features/doc_capture_helper.rb

```diff
@@ -46,4 +46,16 @@ def mock_doc_captured(user_id, response = DocAuth::Response.new(success: true))
     user = User.find(user_id)
     user.document_capture_sessions.last.store_result_from_response(response)
   end
+
+  def expect_doc_capture_page_header(text)
```

Comment:
👏🏻 I know you already merged this, but I just wanted to chime in and say that these are helpful helper methods!